pkg/deploy/apply_test.go

```diff
@@ -396,10 +412,64 @@ func TestHandleResourceCompletionEvent(t *testing.T) {
 			},
 			isCompleted:  false,
 			errorRequire: require.Nil,
+		}, {
+			desc:      "Correctly handles externalsecrets completed after the start time",
+			startTime: time.Now(),
+			res:       extsec,
+			event: &watch.Event{
+				Type: watch.Modified,
+				Object: &unstructured.Unstructured{
+					Object: map[string]interface{}{
+						"metadata": map[string]string{
+							"name": "extsec-name",
+						},
+						"status": map[string]interface{}{
+							"refreshTime": time.Now().Add(time.Hour).UTC().Format(time.RFC3339),
```

Comment:
Can we test with empty refreshTime? It should have the zero value

Reply:
:+1: